### PR TITLE
Symfony 5.4

### PR DESCRIPTION
**Pull request by rimi-itk**
Upgrades to Symfony 5.4

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,55 +34,55 @@
...
+        "symfony/notifier": "5.4.*",
+        "symfony/process": "5.4.*",
+        "symfony/property-access": "5.4.*",
+        "symfony/property-info": "5.4.*",
+        "symfony/proxy-manager-bridge": "5.4.*",
+        "symfony/security-bundle": "5.4.*",
+        "symfony/serializer": "5.4.*",
+        "symfony/string": "5.4.*",
+        "symfony/translation": "5.4.*",
+        "symfony/twig-bundle": "^5.3",
```
Comment (lats1) [inline composer.json:65]: Shouldn't this also be `5.4.*`?

File: composer.json [hunk 2]
```diff
@@ -34,55 +34,55 @@
...
+        "symfony/debug-bundle": "^5.3",
         "symfony/maker-bundle": "^1.0",
-        "symfony/phpunit-bridge": "^5.2",
-        "symfony/stopwatch": "^5.2",
-        "symfony/var-dumper": "^5.2",
-        "symfony/web-profiler-bundle": "^5.2"
+        "symfony/phpunit-bridge": "^5.3",
+        "symfony/stopwatch": "^5.3",
+        "symfony/var-dumper": "^5.3",
+        "symfony/web-profiler-bundle": "^5.3"
```
Comment (lats1) [inline composer.json:85]: I get that they probably will have the right version, but shouldn't we change the constraints to `^5.4`?

**Review by lats1 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (rimi-itk) [reply composer.json]: I've changed to `5.4.*` in https://github.com/itk-dev/naevnssekretariatet/pull/82/commits/886511f8eaf5f0e8e2861ef4d734d44010f4158f.

File: composer.json (hunk 1 above)
Comment (rimi-itk) [reply composer.json]: Yes. Fixed in https://github.com/itk-dev/naevnssekretariatet/pull/82/commits/886511f8eaf5f0e8e2861ef4d734d44010f4158f.

**Review by lats1 — APPROVED** [review]

**Pull request merged**